src/commands/toTheGallows.ts

```diff
@@ -0,0 +1,30 @@
+import { SlashCommandBuilder } from 'discord.js';
+import { buildEvilHangmanMessage } from '../evilHangman/evilHangmanMessage';
+import { EvilHangmanGame } from '../evilHangman/evilHangmanGame';
+
+const LengthOption = 'wordlength';
+const GuessesOption = 'numguesses';
+
+const builder = new SlashCommandBuilder()
+	.setName('tothegallows')
+	.setDescription('Begins a new game of Evil Hangman')
+	.addIntegerOption(option =>
+		option.setName(LengthOption).setDescription('The number of letters in the word to guess')
+	)
+	.addIntegerOption(option =>
+		option.setName(GuessesOption).setDescription('The number of allowed incorrect guesses')
+	);
```

Comment:
Good call, done. Also I changed the format of UserMessageErrors a bit

Reply:
Glad we check that in the business logic, but if the limits are known, we should probs inform Discord about that too, with `setMaxValue` and `setMinValue`

Reply:
neat, didn't know we could do that. I didn't include a max for word length, since the valid options there depend on the contents of the dictionary, but I don't think we're planning on having negative length words any time soon.

Reply:
Fair enough. The dictionary should in theory be easy to query, but that can be a future improvement.